registries/adapters/local_openshift_adapter.go

```diff
@@ -69,87 +71,55 @@ func (r LocalOpenShiftAdapter) GetImageNames() ([]string, error) {
 		return nil, err
 	}
 
-	images, err := openshiftClient.ListRegistryImages()
-	if err != nil {
-		log.Errorf("Failed to load registry images")
-		return nil, err
+	imageClient := openshiftClient.Image()
+	if r.Config.Namespaces == nil {
+		log.Debug("Didn't find any namespace in broker configuration, assuming `openshift`.")
```

Comment:
Is it appropriate to use the term "broker configuration" here, since bundle-lib isn't necessarily tied to a broker?

Reply:
I would just remove broker from that line.